src/dotnet-core-uninstall/Shared/Configs/BundleTypePrintInfo.cs

```diff
@@ -4,19 +4,21 @@
 using Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo;
 using Microsoft.DotNet.Tools.Uninstall.Shared.BundleInfo.Versioning;
 
-namespace Microsoft.DotNet.Tools.Uninstall.Shared.Commands
+namespace Microsoft.DotNet.Tools.Uninstall.Shared.Configs
 {
     internal abstract class BundleTypePrintInfo
     {
         public abstract BundleType Type { get; }
 
         public string Header { get; }
         public Func<IList<Bundle>, GridView> GridViewGenerator { get; }
+        public string OptionName { get; }
```

Comment:
nit: maybe something like commandline option. It sounds like the "name" is optional

Reply:
I think the name `Option` has a very specific meaning throughout the entire repository; it means the command line option. Hence I still feel like `OptionName` is clear enough for this field.